idle-rpg/game/utils/Event.js

```diff
@@ -140,6 +146,10 @@ class Event {
                 const eventLog = `Killed ${randomPlayer.name} in ${selectedPlayer.map.name}.`;
                 const otherPlayerLog = `Died to ${selectedPlayer.name} in ${selectedPlayer.map.name}.`;
 
+                if (randomPlayer.horse = 0){
```

Comment:
in line 93 you used
`selectedPlayer.horse === 'yes'`
You're comparing it differently here